Cargo.toml

```diff
@@ -23,7 +23,7 @@ capnp-rpc = "0.18.0"
 capnpc = "0.18.0"
 chrono = "0.4.31"
 clap = { version = "4.4.4", features = ["derive", "env"] }
-ethers = { git = "https://github.com/avichalp/ethers-rs.git", branch = "avichalp/calibrationnet" }
```

Comment:
Looks like the branch `avichalp/calibrationnet` no longer exists, and the change was merged. We can safely use original repo